models/tasks.js

```diff
@@ -667,6 +667,28 @@ const updateOrphanTasksStatus = async (lastOrphanTasksFilterationTimestamp) => {
   }
 };
 
+const markUnDoneTasksOfArchivedUsersBacklog = async (users) => {
```

Comment:
please use Use Batch Writes: To improve efficiency and consistency, consider using a Firestore batch write instead of updating documents within the loop